crashtracker-ffi/src/crash_info/builder.rs

```diff
@@ -313,9 +313,13 @@ pub unsafe extern "C" fn ddog_crasht_CrashInfoBuilder_with_stack(
 #[named]
 pub unsafe extern "C" fn ddog_crasht_CrashInfoBuilder_with_thread(
     mut builder: *mut Handle<CrashInfoBuilder>,
-    thread: ThreadData,
+    mut thread: ThreadData,
 ) -> VoidResult {
     wrap_with_void_ffi_result!({
+        if thread.crashed {
+            let stack = (thread.stack.to_inner_mut())?.clone();
+            builder.to_inner_mut()?.with_stack(stack)?;
```

Comment:
I wonder if `ddog_crasht_CrashInfoBuilder_with_stack` shouldn't be removed ? 🤔

Reply:
If there is only one executing thread, does it make sense to have it twice?

Reply:
But you're the main user of this API, so do what makes sense to you!